pkg/log/log_test.go

```diff
@@ -7,16 +7,16 @@ import (
 	"testing"
 	"time"
 
-	"github.com/sirupsen/logrus"
+	"log/slog"
```

Comment:
The imports are not formatted correctly.

Reply:
please can you explain again what you meant here, can't understand as to how I should format it. 

I mean is it the line spacing in between?

Reply:
Import should be formatted:
```
import (
  <std imports>

  <external imports>
)
```